### ISSUE TITLE
Installing bootable Debian sid fails with `depmod: 1: exec: depmod: not found`

### ISSUE THREAD

**Issue by paulmenzel** [systemd/mkosi#2584]
I added *systemd-boot* and *kmod* explicitly, although, *kmod* is installed by default, so `/usr/sbin/depmod` should be in the image.

    $ git log --no-decorate --oneline -1
    c824325c Merge pull request #2572 from DaanDeMeyer/journal
    $ more mkosi.local.conf
    [Content]
    #Repositories=main,universe
    Packages=linux-image-cloud-amd64
             systemd
             systemd-boot
             kmod
             udev
             dbus
    $ mkosi -d debian -r sid
    […]
    Setting up file (1:5.45-3) ...
    Setting up kmod (32-1) ...
    sed: can't read /etc/modules: No such file or directory
    Setting up libassuan0:amd64 (2.5.6-1) ...
    Setting up libgomp1:amd64 (14-20240330-1) ...
    Setting up libfdt1:amd64 (1.7.0-2) ...
    Setting up amd64-microcode (3.20240116.2) ...
    amd64-microcode: initramfs support missing
    Setting up libexecs1:amd64 (1.4-2) ...
    Setting up libunwind8:amd64 (1.6.2-3) ...
    Setting up libsemanage-common (3.5-1) ...
    Setting up libsource-highlight-common (3.1.9-4.3) ...
    Setting up libsframe1:amd64 (2.42-4) ...
    Setting up libelf1t64:amd64 (0.191-1) ...
    Setting up libjansson4:amd64 (2.14-2+b2) ...
    Setting up libfsverity0:amd64 (1.5-1.1+b2) ...
    Setting up libeac3:amd64 (1.1.2+ds+git20220117+453c3d6b03a0-1.1+b2) ...
    Setting up libkrb5support0:amd64 (1.20.1-6+b1) ...
    Setting up libdw1t64:amd64 (0.191-1) ...
    Setting up systemd-boot (255.4-1+b1) ...
    Setting up ovmf (2024.02-2) ...
    Setting up tzdata (2024a-1) ...
    […]
    Setting up kmod (32-1) ...
    sed: can't read /etc/modules: No such file or directory
    […]
    Processing triggers for shim-signed:amd64 (1.40+15.7-1) ...
    ‣  Running prepare script /home/build/src/mkosi/mkosi.prepare.chroot…
    ‣ bootctl is not installed, assuming 'unknown' kernel type
    ‣  Running depmod for 6.7.9-cloud-amd64
    depmod: 1: exec: depmod: not found
    ‣ "depmod --all --basedir /buildroot 6.7.9-cloud-amd64" returned non-zero exit code 127.
    ‣  (Fixing ownership of package manager cache directory)

**Comment by paulmenzel** [comment]
It is installed, but not in the path. Doing `export PATH=/usr/sbin:$PATH`, it is found.

```
[…]
Processing triggers for initramfs-tools (0.142) ...
Processing triggers for shim-signed:amd64 (1.40+15.7-1) ...
‣  Running prepare script /home/build/src/mkosi/mkosi.prepare.chroot…
‣ bootctl is not installed, assuming 'unknown' kernel type
‣  Running depmod for 6.7.9-cloud-amd64
‣  Copying in extra file trees…
‣  Setting up autologin…
‣ bootctl is not installed, assuming 'unknown' kernel type
‣ An EFI bootable image with systemd-boot was requested but bootctl was not found
‣  (Fixing ownership of package manager cache directory)
```

`bootctl is not installed, assuming 'unknown' kernel type` is still printed in red.

**Comment by DaanDeMeyer** [comment]
`bootctl` also needs to be installed on the host.

**Comment by paulmenzel** [comment]
Indeed. After installing *systemd-boot*, I was able to build a Debian sid image. Thank you very much for your support, and your queued up improvements for these things.

No idea, how the `/sbin` not in `PATH` issue can be fixed on Debian systems.

I am going to leave the issue open, as it should be automatically closed when the merge/pull request gets accepted.

**Comment by Cornelicorn** [comment]
> No idea, how the `/sbin` not in `PATH` issue can be fixed on Debian systems.

You should be able to modify your profile or if you just want to mkosi to look in there, you could use `ExtraSearchPaths=`
